catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/CswRecordMapperFilterVisitor.java

```diff
@@ -503,5 +579,26 @@ public Object visit(Function function, Object extraData) {
             return super.visit(function, extraData);
         }
     }
+
+    private static void convertGeometryExpressionToEpsg4326(Expression expression) {
+        if (expression instanceof LiteralExpressionImpl) {
```

Comment:
can't tell if this is better or worse. less complexity yes, but also i dont like the java not `!`. Anyways just posting it for comparison. No changes necessary
```java
if (!(expression instanceof LiteralExpressionImpl)) {
    return;
}

LiteralExpressionImpl literalExpression = (LiteralExpressionImpl) expression;
Object valueObj = literalExpression.getValue();
if (!(valueObj instanceof Geometry)) {
    return;
} 

Geometry geometry = (Geometry) valueObj;
Object userDataObj = geometry.getUserData();
if (!(userDataObj instanceof CoordinateReferenceSystem)) {
    return;
}

CoordinateReferenceSystem sourceCRS = (CoordinateReferenceSystem) userDataObj;
try {
    Geometry convertedGeometry = GeospatialUtil.transformToEPSG4326LonLatFormat(geometry, sourceCRS);
    literalExpression.setValue(convertedGeometry);
} catch (GeoFormatException e) {
    LOGGER.trace("Unable to convert geometry to EPSG:4326 format", e);
}
    
```